resources/shared/easyrpg.ini

```diff
@@ -0,0 +1,51 @@
+[EasyRPG]
+FileFormatVersion=1
+
+[783aa77b/*]
```

Comment:
Could you recheck all the checksums? The LDB matches for me but the LMT is wrong, I get "FF3D8719".
It is very easy to change the LMT by accident: Just opening the editor and leaving with a different Map selected alters the LMT.

Reply:
You're correct! Re-downloaded and checked on the command line (no opening in the editor).